### DISCUSSION TITLE
Does update to progress trigger show updated?

### DISCUSSION THREAD

**Discussion by mmoore99** [trakt/trakt-api#440] [Q&A] [answered]
When accessing https://api.trakt.tv/shows/updates/start_date do updates to a show's progress trigger inclusion n this call? Specifically, if a new episode for a show airs does this trigger inclusion?

**Comment by rectifyer** [comment, marked as answer]
No, it requires the show metadata to be updated. Presumably you're already caching the show and episode data and would be able to use that already in your app.
